theme.config.tsx

```diff
@@ -1,6 +1,40 @@
+import { useRouter } from 'next/router';
+
+const URL = 'https://github.com/approvers/docs';
+
+// TODO: ロゴを設定する
+/**
+ * nextra のテーマコンフィグファイル
+ * https://nextra.site/docs/docs-theme/theme-configuration
+ */
 export default {
   logo: <span>approvers/docs</span>,
+  docsRepositoryBase: `${URL}/pages`,
   project: {
-    link: 'https://github.com/approvers/docs'
+    link: URL
+  },
+  editLink: {
+    text: 'このページを編集する'
+  },
+  navigation: {
+    prev: true,
+    next: true
+  },
+  feedback: {
+    content: 'issueを提出する',
```

Comment:
```suggestion
    content: 'Issueを提出する',
```
表示を見てみたところ, 大文字のほうがよいかと